distributed/client.py

```diff
@@ -1181,6 +1198,9 @@ async def _start(self, timeout=no_default, **kwargs):
         for topic, handler in Client._default_event_handlers.items():
             self.subscribe_topic(topic, handler)
 
+        for preload in self.preloads:
+            await preload.start()
```

Comment:
In addition, we'll also want to make sure to call the preload teardown methods when the client closes down. Similar to what we do in the worker here

https://github.com/dask/distributed/blob/9fdada1fef36545ef5b560a512d21709d7d46b56/distributed/worker.py#L1615-L1616

Reply:
I was wondering why testing for the teardown method didn't work when I added it before 🤦